### DISCUSSION TITLE
Bounty notification: MFEK Fndn Inc. multi-issue bounty offered for issues related to the CJK stack

### DISCUSSION THREAD

**Discussion by ctrlcctrlv** [sile-typesetter/sile#1773] [General]
This is a multi-issue bounty, the end goal of which is to fix and regularize
SILE's handling of things important to CJK typesetting.

MFEK Foundation Inc. will pay $5,000 to the person whose work closes the main
issue #1245, by closing:

* #1245
* #1190
* #1402
* #918
* #917
* #501
* #1047

These issues should not be solved piecemeal, as #1245 says, the entire stack is
upside down. A rewrite is necessary. Therefore, it will either be paid to one
party (or multiple parties split by some equitable method to be determined at
@ctrlcctrlv's discretion if multiple people contribute to the large patch
closing all issues).

Signed,
Fredrick R. Brennan
MFEK Foundation Inc.
10 April 2023

<details>
<summary>GnuPG block</summary>

```
-----BEGIN PGP SIGNED MESSAGE-----
Hash: SHA256

This is a multi-issue bounty, the end goal of which is to fix and regularize
SILE's handling of things important to CJK typesetting.

MFEK Foundation Inc. will pay $5,000 to the person whose work closes the main
issue #1245, by closing:

* #1245
* #1190
* #1402
* #918
* #917
* #501
* #1047

These issues should not be solved piecemeal, as #1245 says, the entire stack is
upside down. A rewrite is necessary. Therefore, it will either be paid to one
party (or multiple parties split by some equitable method to be determined at
@ctrlcctrlv's discretion if multiple people contribute to the large patch
closing all issues).

Signed,
Fredrick R. Brennan
MFEK Foundation Inc.
10 April 2023
-----BEGIN PGP SIGNATURE-----

iHUEARYIAB0WIQS1rLeeEfG/f0nzK7hYUwVpYvFOWAUCZDSWfQAKCRBYUwVpYvFO
WH83AP9dOTqhAfqM0jY9tm6nmA/UdGWAwm87s9KoMtLwCo5DmwD/c2QypG34DoCz
5VUGjOll1blza94Qy5junTljZ1PCWg4=
=vKjg
-----END PGP SIGNATURE-----
```

</details>

**Comment by alerque** [comment]
This is fantastic, thanks for supporting driving this forward!

I've promoted this set of issues [to a project](https://github.com/orgs/sile-typesetter/projects/8) just for another view of their status.
